algorithms/warmup/DiagonalDifference.php

```diff
@@ -0,0 +1,75 @@
+<?php
+echo "hello world";
+
+// function DiagonalDifference($arr)
```

Comment:
Always ensure unit tests pass, currently they fail with

**1) Error
The data provider specified for DiagonalDifferenceTest::test_DiagonalDifference is invalid.
PHPUnit\Util\Exception: The data set for the @testWith annotation cannot be parsed: Syntax error**
at https://github.com/julacha/HackerRank/pull/1/checks

In this case, they fail due to uncommented function and temporary changes in this file.